pkg/controller/quay_io_ci_images_distributor/quay_io_ci_images_distributor.go

```diff
@@ -245,7 +245,7 @@ func testInputImageStreamTagFilterFactory(
 	l = logrus.WithField("subcomponent", "test-input-image-stream-tag-filter")
 	return func(nn types.NamespacedName) bool {
 		if ignoreImageStreamTags.Has(nn.String()) {
-			logrus.WithField("tag", nn.String()).Fatal("Ignored events of image stream tag")
```

Comment:
Oh Boy!
This is a very bad one from a temporary code for testing.
It explains lots of things.